kasa/experimental/smartcameramodule.py

```diff
@@ -0,0 +1,95 @@
+"""Base implementation for SMART modules."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from ..exceptions import DeviceError, KasaException, SmartErrorCode
+from ..smart.smartmodule import SmartModule
+
+if TYPE_CHECKING:
+    from .smartcamera import SmartCamera
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SmartCameraModule(SmartModule):
+    """Base class for SMARTCAMERA modules."""
+
+    NAME: str
+
+    #: Query to execute during the main update cycle
+    QUERY_GETTER_NAME: str
+    QUERY_MODULE_NAME: str
+    QUERY_SECTION_NAMES: str | list[str]
```

Comment:
I'd document these class vars, too, to make it easier to understand the data model.

Reply:
Done